src/engraving/rw/inoutdata.h

```diff
@@ -46,6 +46,7 @@ struct ReadInOutData {
     // for except - in
     ReadLinks links;
     std::optional<double> overridedSpatium = std::nullopt;
+    std::optional<double> savedSpatium = std::nullopt;
```

Comment:
Seems like this would rather belong under "out" (instead of "in"), doesn't it?

Reply:
yep =) fixed